testsrc/jstests/harmony/math-functions.js

```diff
@@ -54,4 +54,78 @@ assertEquals(Math.imul(-2, -2), 4);
 assertEquals(Math.imul(0xffffffff, 5), -5);
 assertEquals(Math.imul(0xfffffffe, 5), -10);
 
+assertEqualsDelta(Math.atanh(1/2), 0.549306144334059, 0.0000000000001);
```

Comment:
because the 262 tests suite seems to have no tests for this maybe you can add some more samples

Reply:
Done